plugins/inputs/intel_powerstat/intel_powerstat.go

```diff
@@ -41,6 +41,7 @@ const (
 type PowerStat struct {
 	CPUMetrics     []string        `toml:"cpu_metrics"`
 	PackageMetrics []string        `toml:"package_metrics"`
+	ReadTimeout    int64           `toml:"msr_read_timeout"`
```

Comment:
`msr_read_timeout` -> `msr_read_timeout_ms` ?

Reply:
Not relevant if this will become a `config.Duration`.